open-liberty/content.md

```diff
@@ -94,6 +100,68 @@ The images are designed to support a number of different usage patterns. The fol
 	  --volumes-from app %%IMAGE%%:webProfile8
 	```
 
+# Using `springBoot` images
+
+The `springBoot` images introduce capabilities specific to the support of Spring Boot applications, including the `springBootUtility` used to separate Spring Boot applications into thin applications and dependency library caches. To elaborate these capabilities this section assumes the standalone Spring Boot 2.0.x application `hellospringboot.jar` exists in the `/tmp` directory.
+
+1.	A Spring Boot application JAR deploys to the `dropins/spring` directory within the default server configuration, not the `dropins` directory. Liberty allows one Spring Boot application per server configuration. The following example starts a container running a Spring Boot application.
+
+	```console
+	$ docker run -d -p 8080:9080 \
+	    -v /tmp/hellospringboot.jar:/config/dropins/spring/hellospringboot.jar \
+	    %%IMAGE%%:springBoot2
+	```
+
+	Similarly, you can create a Spring Boot application layer over this image by adding the application JAR to the `dropins/spring` directory. In this example we copied `hellospringboot.jar` from `/tmp` to the same directory containing the following Dockerfile.
+
+	```dockerfile
+	FROM %%IMAGE%%:springBoot2
+	ADD  hellospringboot.jar /config/dropins/spring/
```

Comment:
This should be `COPY` instead of `ADD` (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy). `ADD` should only be recommended when extracting a local tar file.

 (should be updated in `websphere-liberty` as well)